s3/replication/common/src/s3replicationcommon/job.py

```diff
@@ -41,6 +43,9 @@ class JobState(Enum):
     ABORTED = 6  # explicitly aborted
     FAILED = 7
 
+    def __str__(self):
+        return self.name
```

Comment:
I guess, __str__ we use when we want to print object string. we don't have any instance variable called name. is this correct way to get caller object string using __str__ ?

Reply:
https://docs.python.org/3/library/enum.html#allowed-members-and-attributes-of-enumerations
Yes, this is to convert to string, for example str(JobState.RUNNING) should return "RUNNING" instead of JobState.RUNNING